apis/stash/v1beta1/restore_session_types.go

```diff
@@ -75,6 +76,10 @@ type RestoreSessionSpec struct {
 	// Don't specify it in sidecar model.
 	// +optional
 	InterimVolumeTemplate *core.PersistentVolumeClaim `json:"interimVolumeTemplate,omitempty" protobuf:"bytes,8,opt,name=interimVolumeTemplate"`
+	// Actions that Stash should take in response to restore sessions.
+	// Cannot be updated.
```

Comment:
`// Cannot be updated.` what does this mean?

Reply:
I have copy pasted from BackupConfiguration. You had added that line. I don't know why. Should I remove it?